dlt/common/utils.py

```diff
@@ -504,6 +504,19 @@ def without_none(d: Mapping[TKey, Optional[TValue]]) -> Mapping[TKey, TValue]:
     return {k: v for k, v in d.items() if v is not None}
 
 
+def exclude_keys(d: Mapping[str, Any], keys: Iterable[str]) -> Dict[str, Any]:
```

Comment:
just a small suggestion: if we're moving this code to the core, maybe we should change the name of the first arg from `d` to something more meaningful? :) Like `data_dict` or `input_dict`

Reply:
This is a very good point, thanks for pointing it out @VioletM, I named it `d` for consistency with some of the other functions from `dlt.common.utils`, e.g. `dict_remove_nones_in_place` or `without_none`. 
But if we look at how Python itself calls that argument, it should be called `mapping` instead since it operates on a mapping type. (examples: https://docs.python.org/3/library/stdtypes.html#dict, https://docs.python.org/3/library/stdtypes.html#str.format_map)
I'll update the PR